packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -15,37 +15,142 @@
 
 package com.amazonaws.amplify.amplify_datastore
 
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
 import androidx.annotation.NonNull
-
+import androidx.annotation.VisibleForTesting
+import com.amazonaws.amplify.amplify_datastore.types.FlutterDataStoreFailureMessage
+import com.amazonaws.amplify.amplify_datastore.types.model.FlutterModelSchema
+import com.amazonaws.amplify.amplify_datastore.types.model.FlutterSerializedModel
+import com.amazonaws.amplify.amplify_datastore.types.query.QueryOptionsBuilder
+import com.amazonaws.amplify.amplify_datastore.util.safeCastToList
+import com.amazonaws.amplify.amplify_datastore.util.safeCastToMap
+import com.amplifyframework.core.Amplify
+import com.amplifyframework.core.Consumer
+import com.amplifyframework.core.model.Model
+import com.amplifyframework.core.model.query.QueryOptions
+import com.amplifyframework.core.model.query.predicate.QueryPredicates
+import com.amplifyframework.core.model.temporal.Temporal
+import com.amplifyframework.datastore.AWSDataStorePlugin
+import com.amplifyframework.datastore.DataStoreException
+import com.amplifyframework.datastore.DataStoreItemChange
+import com.amplifyframework.datastore.appsync.SerializedModel
 import io.flutter.embedding.engine.plugins.FlutterPlugin
 import io.flutter.plugin.common.MethodCall
 import io.flutter.plugin.common.MethodChannel
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler
 import io.flutter.plugin.common.MethodChannel.Result
-import io.flutter.plugin.common.PluginRegistry.Registrar
+import java.util.*
```

Comment:
In the past, I've had @jamesonwilliams recommend avoiding "*" imports.

Reply:
yeah I agree. Somehow that was default android studio formatting settings to merge java.util imports (and a couple others only)